src/server/WebDav.ts

```diff
@@ -388,34 +388,39 @@ export default class WebDav {
 
 export function readConfigFile(configFilename: string) {
 	return new Observable<DavOptions>(observer => {
-		const stream = createReadStream(configFilename);
-		let chunks: Buffer[] = [];
-
-		// Listen for data
-		stream.on('data', chunk => {
-			chunks.push(chunk);
-		});
+		if (configFilename.match(/\.js$/)) {
+			observer.next(require(configFilename));
```

Comment:
Hi @Eneris 
Thank you for submitting PR. I'll merge PR but need small adjustment here. I as user definitely make some typo or/and other mistake in `dw.js`, So would be great to handle this case, let's say via, `try ... catch`.

Also there are small disadvantage of using `require`: it caches module until app is restarted, so if you change credentials VS Code may not see it until restart. I'm fine with this and we may figureout something later.

Reply:
* Added `try ... catch`
* Invalidating module cache in every request